src/dlgIRC.cpp

```diff
@@ -1,6 +1,8 @@
 /***************************************************************************
+ *   Copyright (C) 2008-2017 The Communi Project                           *
  *   Copyright (C) 2008-2013 by Heiko Koehn - KoehnHeiko@googlemail.com    *
  *   Copyright (C) 2014 by Ahmed Charles - acharles@outlook.com            *
+ *   Copyright (C) 2017 by Fae - itsthefae@gmail.com                       *
```

Comment:
Fair enough, though you might see my real name that isn't my real photo in my avatar...!